### PR TITLE
WIP: Build composer support module

### PR DESCRIPTION
**Pull request by dawehner**

### CODE REVIEW

File: .circleci/composer-admin_ui_support.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "justafish/drupal-admin-ui-support",
```
Comment (tedbow) [inline .circleci/composer-admin_ui_support.json]: This repo doesn't exist anymore does it. Isn't this the module we now have in this repo?

Comment (dawehner) [reply .circleci/composer-admin_ui_support.json]: I thought about building a composer package automatically and push it to https://github.com/jsdrupal/drupal-admin-ui-support-dist, just we do for the build JS

Comment (tedbow) [reply .circleci/composer-admin_ui_support.json]: ok. I don't know how this works but should it be "jsdrupal/" not "justafish/"

Comment (dawehner) [reply .circleci/composer-admin_ui_support.json]: lol yeah we had a discussion about that ¯\_(ツ)_/¯

**Comment by dawehner** [comment]
https://github.com/jsdrupal/drupal-admin-ui/pull/30#pullrequestreview-105744304 replaced this issue

**Pull request closed without merging**